src/components/track.tsx

```diff
@@ -3,49 +3,84 @@ import * as React from "react";
 import { accentColor } from "../constants";
 import { usePublicFileURL } from "../hooks/use-public-file-url";
 
+const TitleContainer = styled.div`
+  position: absolute;
+
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4.5rem;
+  overflow: hidden;
+  transition: 0.5s ease;
```

Comment:
I'd prefer to use

```css
transition-property: height;
transition-duration: ${transitionDuration};
```

instead, so we explicitly know which properties are animated and use the same transition duration as everywhere else

Reply:
Resolved in following commit (i hope, your wording makes it hard for me to tell what you'd acutally like to see changed)